site/en/docs/extensions/reference/alarms/index.md

```diff
@@ -65,6 +65,15 @@ chrome.alarms.onAlarm.addListener((alarm) => {
 });
 ```
 
```

Comment:
@oliverdunk NIT: Could we add these items under a " ## Concepts and usage" section above the examples version? We are trying to follow this pattern. Example [ReadingList concepts](https://developer.chrome.com/docs/extensions/reference/readingList/#concepts),  [DNR concepts](https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/#concepts-and-usage)

Reply:
Done 😄